codegens/http/lib/util.js

```diff
@@ -168,7 +168,8 @@ function getHeaders (request) {
     headers = '';
 
   if (contentTypeIndex >= 0) {
-    if (request.headers.members[contentTypeIndex].value === 'multipart/form-data' || request.body.mode === 'formdata') {
+    if (request.headers.members[contentTypeIndex].value === 'multipart/form-data' ||
+      (request.bidy && request.body.mode === 'formdata')) {
```

Comment:
What is bidy?

Reply:
How is the test passing with this?

Reply:
Sorry. Corrected the typo. and the test is for the snippet generation should not fail if the request does not have a body. This is happening, the snippet is getting generated.